narwhals/_arrow/namespace.py

```diff
@@ -74,7 +74,7 @@ def _create_series_from_scalar(self, value: Any, series: ArrowSeries) -> ArrowSe
         from narwhals._arrow.series import ArrowSeries
 
         if self._backend_version < (13,):  # pragma: no cover
-            value = value.as_py()
+            value = value.as_py() if hasattr(value, "as_py") else value
```

Comment:
I am unsure in which cases the value have `as_py` method. From `null_count` we get an int and therefore this line was breaking.